Makefile

```diff
@@ -1,24 +0,0 @@
-ROOT=$(shell pwd)
-
-test: test-unit test-integration
-
-test-unit:
-	@echo "\nRunning unit tests..."
-	@NODE_ENV=test mocha test/integration test/structure test/support test/unit --recursive
-
-test-integration:
-	@echo "\nRunning integration tests..."
-	rm -rf node_modules/waterline-adapter-tests/node_modules/waterline;
-	ln -s $(ROOT) node_modules/waterline-adapter-tests/node_modules/waterline;
```

Comment:
From looking at `test/adapter/runner.js` it's not clear to me how you ensure that the integration tests run against waterline **edge**. `waterline-adapter-tests` will have as dependency waterline **stable** and this step ensures we are testing against any changes done in waterline.